examples/jvm/src/main/kotlin/Main.kt

```diff
@@ -65,4 +80,21 @@ fun main() = application {
 		dirChosen = path ?: "none selected"
 		showDirPicker = false
 	}
+
+	SaveFilePicker(
+		showSaveFile,
+		filename = "newFile.txt",
+	) { selectedFile ->
+		val contents = "this is a new test file"
+		hasSavedFiled = selectedFile?.file?.let { file ->
+			writeToFile(file, contents)
+			true
+		} ?: false
```

Comment:
```suggestion
		} ?: false
		showSaveFile = false
```

Reply:
I can't open any dialog more than once. that is actually something I wanted to take a look at to see if it's our code or if it's the example compose code that does something weird.

Reply:
I have fixed this. You should be able to open the dialog more than once now

Reply:
nice